### PR TITLE
OSDOCS-3451: Sort Node Updates by Zone and Age

### PR DESCRIPTION
**Pull request by mburke5678**
https://issues.redhat.com/browse/OSDOCS-3451

MCO to upgrade nodes per [topology.kubernetes.io/zone](http://topology.kubernetes.io/zone) and then by node age (oldest first).

Preview: 
[Machine config overview](https://deploy-preview-45261--osdocs.netlify.app/openshift-enterprise/latest/post_installation_configuration/machine-configuration-tasks.html#machine-config-overview-post-install-machine-configuration-tasks), bullet 3
[About the OpenShift Update Service](https://deploy-preview-45261--osdocs.netlify.app/openshift-enterprise/latest/architecture/control-plane.html#update-service-overview_control-plane), paragraph 5

### CODE REVIEW

**Comment by mburke5678** [comment]
@kikisdeliveryservice Can you PTAL at my PR for this change? I should think just a quick description of the upgrade process should be sufficient. 
Thank you in advance.

File: modules/machine-config-overview.adoc [hunk 1]
```diff
@@ -18,6 +18,8 @@ The Machine Config Operator (MCO) manages updates to systemd, CRI-O and Kubelet,
 A node can have multiple labels applied that indicate its type, such as `master` or `worker`, however it can be a member of only a *single* machine config pool.
 ====
 
+* After a machine config change, the MCO updates the affected nodes alphabetically by zone, based on the `topology.kubernetes.io/zone` label. If a zone has more than one node, the oldest nodes are updated first. For nodes that do not use zones, such as in baremetal deployments, the nodes are upgraded by age, with oldest nodes updated first. The MCO updates the number of nodes as specified by the `maxUnavailable` field on the machine configuration pool as a time.
```
Comment (kikisdeliveryservice) [inline modules/machine-config-overview.adoc:21]:
```suggestion
* After a machine config change, the MCO updates the affected nodes alphabetically by zone, based on the `topology.kubernetes.io/zone` label. If a zone has more than one node, the oldest nodes are updated first. For nodes that do not use zones, such as in baremetal deployments, the nodes are upgraded by age, with oldest nodes updated first. The MCO updates the number of nodes as specified by the `maxUnavailable` field on the machine configuration pool at a time. 
```

**Review by kikisdeliveryservice — COMMENTED** [review]
One small typo, but otherwise lgtm!

**Comment by mburke5678** [comment]
@sergiordlr PTAL. Thank you very much!

**Comment by sergiordlr** [comment]
hello @mburke5678! It looks good to me.

Thank you very much!

File: modules/machine-config-overview.adoc [hunk 2]
```diff
@@ -18,6 +18,8 @@ The Machine Config Operator (MCO) manages updates to systemd, CRI-O and Kubelet,
 A node can have multiple labels applied that indicate its type, such as `master` or `worker`, however it can be a member of only a *single* machine config pool.
 ====
 
+* After a machine config change, the MCO updates the affected nodes alphabetically by zone, based on the `topology.kubernetes.io/zone` label. If a zone has more than one node, the oldest nodes are updated first. For nodes that do not use zones, such as in baremetal deployments, the nodes are upgraded by age, with oldest nodes updated first. The MCO updates the number of nodes as specified by the `maxUnavailable` field on the machine configuration pool at a time.
```
[the remark this answers is not in the document]
Comment (Amrita42) [reply modules/machine-config-overview.adoc:21]: I would suggest add **the **  " with **the** oldest nodes..."

**Comment by kikisdeliveryservice** [comment]
Thanks all!!!! :smiley_cat:

**Pull request merged**